src/window/glfw/Cargo.toml

```diff
@@ -28,6 +28,11 @@ documentation = "https://docs.rs/gfx_window_glfw"
 name = "gfx_window_glfw"
 
 [dependencies]
-glfw = "0.13"
+glfw = "0.14"
+gfx = "0.15"
 gfx_core = { path = "../../core", version = "0.7" }
 gfx_device_gl = { path = "../../backend/gl", version = "0.14" }
+
+[[example]]
+name = "window"
+path = "examples/window/main.rs"
```

Comment:
let's simplify it by placing the example as `examples/window.rs`, so that you don't even need this `path=` line here